### ISSUE TITLE
Ts-aside with TS-UI

### ISSUE THREAD

**Issue by CarloAl** [Tradeshift/elements#161]
I am using in my application a mix of TS UI and elements (the recommendation I have received is to use elements and fall back on TS UI when the component is not available in elements). 
If I do not place the `ts-aside` right next to the `data-ts="Main"` the aside is not rendered correctly. 
![image](https://user-images.githubusercontent.com/5701042/65670045-e4a61b80-e044-11e9-8e35-22f98c877322.png)

**Comment by esmailpour-hosein** [comment]
As we talked about it, it's possible to keep the `ts-aside` next to the `data-ts="Main"` if it's needed.
For future reader: you can just have on `<ts-aside>` and change the content from other places. 
For example in React you can have it in the `Main` or whatever it's called and have single state for the current active aside content which show an specific aside content based on your needs. Just remember to check if you actually needing to implement the aside yourself and it's not already implemented in `Panel.js` 👍

**Comment by CarloAl** [comment]
I agree it is possibile, but not very convenient. I think it would be much easier if you can create a `ts-aside` anywhere in your code disregarding of where you are. I had a phone conversation with @zdlm and I think he was going to have a look into it when he had the time. It's a bit ugly to have in the `Main` the declaration of all the aside of the application.
